RELEASE_NOTES.md

```diff
@@ -45,9 +45,21 @@ v1.0.0-alpha.X
   be subclassed.
   [#331](https://github.com/OpenAssetIO/OpenAssetIO/issues/331)
 
+- Removed `Context.managerOptions`.
+  [#291](https://github.com/OpenAssetIO/OpenAssetIO/issues/291)
+
+- Changed `Context.kOther` to `kUnknown`, and changed the default
+  context access to `kUnknown`. This better describes its use, and
+  encourages hosts to properly configure the context before use.
+
```

Comment:
[fix] We also need a note here on the change of `Context.access` to an enum from a `string`.